### PR TITLE
Trailing whitespace encouraged

### PR DESCRIPTION
**Pull request by tychonievich**
One possible solution to #91

### CODE REVIEW

File: specification/gedcom.md [hunk 1]
```diff
@@ -316,6 +316,21 @@ Each non-pointer payload may be encoded in 1 or more line strings (line continua
...
+may be encoded either
+with an empty `LineVal` (i.e. with a space after the `Tag`)
+or with no `LineVal` (i.e. without a space after the `Tag`).
+In both cases, it is recommended that the space be omitted.
+
+:::example
+The payload of a `MARR` structure has type `[Y|<NULL>]`, which is optional but cannot be the empty string.
+The payload of a `EVEN` structure has type `Text`, and `Text` can be the empty string.
+The no-payload "`1 MARR`" should be represented without a trailing space, although including a single trailing space is permitted.
+The empty-payload "`1 EVEN `" should be represented with a single trailing space, although omitting it is permitted.
```
Comment (dthaler) [inline specification/gedcom.md:331]: This line appears to contradict line 325.  Either way, I don't prefer encouraging trailing whitespace.

File: specification/gedcom.md [hunk 2]
```diff
@@ -316,6 +316,21 @@ Each non-pointer payload may be encoded in 1 or more line strings (line continua
 The exact encoding of non-pointer payloads is dependent on the datatype of the payload, as determined by the structure type.
 The datatype of non-pointer payloads cannot be fully determined by line value content alone.
 
+Because empty payloads and missing payloads are considered equivalent,
+both a structure with no payload
+and a structure with the empty string as its payload
+may be encoded either
+with an empty `LineVal` (i.e. with a space after the `Tag`)
+or with no `LineVal` (i.e. without a space after the `Tag`).
+In both cases, it is recommended that the space be omitted.
```
Comment (tychonievich) [inline specification/gedcom.md:325]:
```suggestion
It is recommended that a `LineVal` be included if and only if the structure has a `payload`.
```

File: specification/gedcom.md (hunk 1 above)
Comment (tychonievich) [reply specification/gedcom.md:331]: Oops! Trying to write both this and its other alternative (#93) at once, I appear to have used the wrong line 325.

**Comment by tychonievich** [comment]
Inconsistent with spec; see #91

**Pull request closed without merging**